.ci/msys_test.bat

```diff
@@ -0,0 +1,7 @@
+:: FIXME(equalsraf) currently not all tests work on Windows
+:: for now just run what we can
+mingw32-make functionaltest TEST_FILE=test\functional\eval\printf_spec.lua VERBOSE=1 || goto :error
```

Comment:
>Ah the other culprit seems to be luajit crashes when running certain test groups

You mean it crashes even when it "runs" a test group in search of requested tagged tests? (I assume `before_each()` won't run if the describe has no tests to be run)

> Problem with using pending() like that is that for it would likely end up being used in most tests. And the extra call to pending() seems expensive.

I suppose it would be a pending per file to start with, which is a lot less expensive than "run" all the describes in search of tagged tests. These would be _removed_ as windows support/coverage improves, not being end up with, except for features/tests we decide to not support on windows, which we are supposed to block on windows using `pending()`.